lib/Echidna/Test.hs

```diff
@@ -52,6 +52,8 @@ checkETest (f, a) = asks getter >>= \(TestConf p s) -> do
 -- still solves that test.
 shrinkSeq :: (MonadRandom m, MonadReader x m, Has TestConf x, MonadState y m, Has VM y, MonadThrow m)
           => SolTest -> [Tx] -> m [Tx]
-shrinkSeq t xs = shorten >>= mapM shrinkTx >>= ap (fmap . flip bool xs) check where
+shrinkSeq t xs = sequence [shorten, shrunk, shrunk'] >>= uniform >>= ap (fmap . flip bool xs) check where
```

Comment:
We can simplify this to just `[shorten, shrunk]` since most IRL uses have small sequences

Reply:
done